src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java

```diff
@@ -41,15 +39,14 @@
 public class AggregatedRssFeedProducer {
 
     public static final String RSS_CACHE = "Aggregated RSS feed cache";
-    private static final String SELF_REL = "self";
-
     @VisibleForTesting
     static final String FEED_DESCRIPTION = "JVMBloggers aggregated feed";
     @VisibleForTesting
     static final String FEED_TITLE = "JVMBloggers";
     @VisibleForTesting
     static final String FEED_TYPE = "atom_1.0";
 
+    private static final String SELF_REL = "self";
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.